modules/core/src/iterators/make-stream/make-dom-stream.ts

```diff
@@ -24,7 +24,7 @@ export function makeStream<ArrayBuffer>(
       // Create a byte stream (enables `Response(stream).arrayBuffer()`)
       // Only supported on Chrome
       // See: https://developer.mozilla.org/en-US/docs/Web/API/ReadableByteStreamController
-      // @ts-expect-error Only Chrome supports byte streams
+      // ts-expect-error Only Chrome supports byte streams
```

Comment:
Why not `@`?

Reply:
Commented out because it was starting to be flaky suddenly generating errors. Not clear why. Should remove it completely if no longer needed.